aether-ui/aether/ui/assets/apps/pipeline/components/DeleteStatus.jsx

```diff
@@ -104,7 +104,18 @@ class DeleteStatus extends Component {
                     <div key={schema}>
                       <i className='fa fa-check mr-2' />
                       <label>
-                        { `${schema} : ${this.props.deleteStatus.schemas[schema].is_deleted ? 'Deleted' : 'Not deleted, used by other mappings'}` }
+                        { schema } :
+                        {
+                          this.props.deleteStatus.schemas[schema].is_deleted
+                            ? <FormattedMessage
+                              id='delete.modal.entity.types.delete'
+                              defaultMessage='Deleted'
+                            />
+                            : <FormattedMessage
+                              id='delete.modal.entity.types.not-delete'
```

Comment:
It's nice to see that when you write a typo it's well spread :joy: 
`deleted` and `not-deleted`